perf/benchmark/configs/istio/telemetryv2_sd_cpu_mem.yaml

```diff
@@ -0,0 +1,19 @@
+# Configuration Set10: CPU and memory with telemetryv2 using NullVM with
+# Stackdriver filter with metrics/topology on.
+telemetry_mode: "v2-stackdriver-nologging-nullvm"
+conn:
+  - 16
+qps:
+  - 10
+  - 100
+  - 500
+  - 1000
+  - 2000
+  - 3000
+duration: 240
+metrics:
+  - cpu
+  - mem
+perf_record: false
+run_serversidecar: true
+run_baseline: true
```

Comment:
@richardwxn @mandarjog @howardjohn Once we got the PR merged in, it means we add extra four groups of tests. Which is (4 min * 6 * 3) * 4 / 60= 4.8 hours extra testing time. Since our periodic job already 14 hours. Is it still ok to add around 5 hours extra time?